src/app/mbs/modules/modules.component.html

```diff
@@ -3,7 +3,7 @@ <h2 class="title">Modules</h2>
 <table class="table table-responsive-sm table-hover table-bordered mbs-list-table" infinite-scroll (scrolled)="onScrollDown()">
   <thead>
     <tr>
-      <th *ngFor="let header of ['ID', 'Name', 'Stream', 'Version', 'Context', 'State']"
+      <th *ngFor="let header of ['ID', 'Name', 'Stream', 'Version', 'Context', 'Scratch', 'State']"
           [attr.id]="header.toLowerCase() + 'Header'" scope="col">
         <a *ngIf="header != 'Context'" [routerLink]="['/modules', {'orderBy': header.toLowerCase(), 'orderDirection': getOrderDirection(header)}]"
```

Comment:
Let's disable sorting on the `Scratch` column since it's not intuitive:
```diff
diff --git a/src/app/mbs/modules/modules.component.html b/src/app/mbs/modules/modules.component.html
index 5d8a21e..475537c 100644
--- a/src/app/mbs/modules/modules.component.html
+++ b/src/app/mbs/modules/modules.component.html
@@ -5,13 +5,16 @@
     <tr>
       <th *ngFor="let header of ['ID', 'Name', 'Stream', 'Version', 'Context', 'Scratch', 'State']"
           [attr.id]="header.toLowerCase() + 'Header'" scope="col">
-        <a *ngIf="header != 'Context'" [routerLink]="['/modules', {'orderBy': header.toLowerCase(), 'orderDirection': getOrderDirection(header)}]"
-            [ngClass]="getArrowClass(header)">
+        <a
+          *ngIf="!['Context', 'Scratch'].includes(header); else noSort"
+          [routerLink]="['/modules', {'orderBy': header.toLowerCase(), 'orderDirection': getOrderDirection(header)}]"
+          [ngClass]="getArrowClass(header)"
+        >
           {{ header }}
         </a>
-        <ng-container *ngIf="header === 'Context'">
+        <ng-template #noSort>
           {{ header }}
-        </ng-container>
+        </ng-template>
       </th>
     </tr>
   </thead>
```

Reply:
My thinking was that it would provide the user a way to see _only_ non-scratch module builds. I question the intuitiveness/usefulness of sorting by Stream.

Reply:
Of course, sorting by the `Scratch` column means keeping it as a column. I'll create an alternate version of this PR using the interface that appends "(scratch)" to the module name just to see how that looks and feels.

Reply:
See PR #10 for the alternate version.